mne/io/ctf/info.py

```diff
@@ -31,11 +37,18 @@ def _pick_isotrak_and_hpi_coils(res4, coils, t):
     n_coil_head = 0
     for p in coils:
         if p['valid']:
+            if p['kind'] in [CTF.CTFV_COIL_LPA, CTF.CTFV_COIL_RPA,
+                             CTF.CTFV_COIL_NAS]:
+                kind = FIFF.FIFFV_POINT_CARDINAL
+                ident = _ctf_to_fiff[p['kind']]
+            else:  # CTF.CTFV_COIL_SPARE
+                kind = FIFF.FIFFV_POINT_HPI
```

Comment:
HPI mark locations of HPI coils on the head, extra points are just any points along the head surface.

Maybe it's the case that the spare are always HPI locations, but in that case you shoud update your dict above. Maybe just remove your entry for the SPARE?